functions/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Do we need separate .gitignore and .eslintrc files for the Firebase functions folder?

If it also works with our global .gitignore and .eslintrc files, I propose that we delete these local ones.